packages/storefront/src/pages/must-know/versioning.md

```diff
@@ -1,15 +1,15 @@
 # Versioning
 
-We value stability and transparency. Thus we strive to ensure that our reusable components, Sketch Libraries and
-familiar practices don't become obsolete often and unexpectedly.
+We value stability and transparency. Thus we strive to ensure that our reusable components, libraries and familiar
+practices don't become obsolete often and unexpectedly.
```

Comment:
I don' reallyget the sense of this sentence 🤓. 
"_...familiar practices don't become obsolete often and unexpectedly_" ?

Maybe we could strip it down to something like this:
"Thus we strive to ensure that our reusable components and libraries don't become outdated and obsolete."